cubic-in-out.glsl

```diff
@@ -1,7 +1,7 @@
 float cubicInOut(float t) {
   return t < 0.5
     ? 4.0 * t * t * t
-    : 0.5 * pow(2.0 * t - 2.0, 3.0) + 1.0;
+    : 1. - 4.0 * pow(1. - t, 3.0);
```

Comment:
What would you think instead of


```suggestion
    : 1. - 4.0 * pow(abs(1. - t), 3.0);
```

or my favorite solution, I think,

```glsl
float cubicInOut(float t) {
  float tm1 = t - 1.0;
  return t < 0.5
    ? 4.0 * t * t * t
    : 1.0 + 4.0 * tm1 * tm1 * tm1;
}
```

For the reasons mentioned in https://github.com/glslify/glsl-easings/issues/7#issuecomment-660246110, namely that even though people *shouldn't* plug in `t < 0` or `t > 1`, they certainly *will*, perhaps won't notice they've done it, and it's at least nice if the returned value isn't formally undefined and platform-dependent (which, in my experience usually gets caught years later when a developer gets a new computer, runs the same code, and suddenly things look all wrong).

Reply:
Thanks for keeping this library up. That makes sense.

I like the tm1 formula with no pows at all better, too. But I like '1-t' instead of 't - 1' because the formulas can stay the same whether it's cubic or quartic, rather than flipping the sign in the final addition. Commits below do it with an explicit if block to avoid an unnecessary subtraction when `t < .5`.